docs/guides/embeds.md

```diff
@@ -0,0 +1,85 @@
+# How to Embed the Video.js player
+
+Video.js is meant to be an ehnacement to the video element in HTML5 so for years, it's embed code has been just a `<video>` element.
+Video.js then wraps the video element in a div that is used for us to place controls and anything else that's required for the player.
+
+For a long time this was enough. In 2016, a "div ingest" which allows the developer to give Video.js a player div to use instead of making it's own.
+This is partly to help with content reflow but also to help with iOS where you sometimes need to prime the video element and we re-create the video element when we create the player div.
+However, this is kind of weird to have a `<video>` element embed with a `<div>` wrapped around it. So, we built out a new embed, a `<video-js>` embed.
+
+Below, the three kinds of embeds are detailed.
+
+## Embeds
+### `<video>` embed
+The classic Video.js embed. You can then initialize it via `data-setup` or via the `videojs` method.
+
+```html
+<!-- via data-setup -->
+<video id="vid1" class="video-js" data-setup='{}'>
+  <source src="//vjs.zencdn.net/v/oceans.mp4">
+</video>
+
+<!-- via code -->
+<video id="vid1" class="video-js">
+  <source src="//vjs.zencdn.net/v/oceans.mp4">
+</video>
+```
+```js
+const player = videojs('vid1', {});
+```
+
+### Player div ingest
+The enhanced classic embed. You can also initialize it via `data-setup` or via `videojs` method.
+
+```html
+<!-- via data-setup -->
+<div data-vjs-player>
+  <video id="vid1" class="video-js" data-setup='{}'>
+    <source src="//vjs.zencdn.net/v/oceans.mp4">
+  </video>
+</div>
+
+<!-- via code -->
+<div data-vjs-player>
+  <video id="vid1" class="video-js">
+    <source src="//vjs.zencdn.net/v/oceans.mp4">
+  </video>
+</div>
+```
+```js
+const player = videojs('vid1', {});
+```
+
+As you can see, it isn't much different from the classic `<video>` embed. It also does make it easier to work with [React](/docs/guides/react.md).
+
+### `<video-js>` embed
+This is the [I Can't Believe It's Not Custom Elements](https://developers.google.com/web/fundamentals/web-components/customelements) embed code.
+It looks very much like the `<video>` embed but instead of `video` it's a `video-js` embed.
+This is useful for all the things that the player div ingest is useful for and it matches our library name!
+
+```html
+<!-- via data-setup -->
+<video-js id="vid1" class="video-js" data-setup='{}'>
```

Comment:
Should you need the `class="video-js"`? Could we add that if it's missing?

Reply:
Could make sense to add it for the `video-js` element. Adding it unconditionally could potentially cause stylistic bugs.